lib/edgedb/error.ex

```diff
@@ -10,6 +10,14 @@ defmodule EdgeDB.Error do
 
     * `EdgeDB.Error.retry?/1`
     * `EdgeDB.Error.inheritor?/2`
+
+  By default the client generates exception messages in full format, attempting to output all possible and
+    useful information about the error location if it is possible.
```

Comment:
Minor nit, possible is used twice here in a way that makes it redundant. Maybe remove one of them?

Reply:
Yeah, you're right, thanks. I fixed it, leaving only the last one